src/sentry/tasks/store.py

```diff
@@ -294,6 +295,25 @@ def retry_process_event(process_task_name: str, task_kwargs: dict[str, Any], **k
     process_task.delay(**task_kwargs)
 
 
+def is_process_disabled(project_id: int, event_id: str, platform: str) -> bool:
+    if killswitch_matches_context(
+        "store.load-shed-process-event-projects",
+        {
+            "project_id": project_id,
+            "event_id": event_id,
+            "platform": platform,
+        },
+    ):
+        return True
+
+    process_project_rollout = options.get("store.load-shed-process-event-projects-gradual")
+    rollout_rate = process_project_rollout.get(project_id)
+    if not rollout_rate:
```

Comment:
This will cause a rollout rate of 0.0 do the same as 1.0. Better to check for is not None to avoid future incidents.

Reply:
no it won't 
the option is a load shed rate, not a rollout rate.
option is null -> return False
option is 0.0 -> return false 
option 1.0 -> return true 

Added a test case to show this

Reply:
Sorry yeah, misread this.